README.md

```diff
@@ -203,14 +203,10 @@ If you want to include only subset of databases, you can use option `--include-d
 
 ### Running as non-superuser
 
-To be able to collect metrics from `pg_stat_activity` and `pg_stat_replication`
-as  non-superuser you have to create functions and views as a superuser, and
-assign permissions separately to those.
-
-In PostgreSQL, views run with the permissions of the user that created them so
-they can act as security barriers. Functions need to be created to share this
-data with the non-superuser. Only creating the views will leave out the most
-important bits of data.
+To be able to collect metrics from `pg_stat*` views as non-superuser in PostgreSQL
+server versions >= 10 you can grant pg_monitor built-in role to the user. If
```

Comment:
"you can grant *the* pg_monitor built-in role" I'd say, maybe also markup pg_monitor with backticks, like `pg_stat*` before.
Also, I guess technically the `pg_read_all_stats` role would suffice, so maybe mention both

Reply:
Fixed. PostgreSQL documentation says that `pg_monitor` is a member of `pg_read_all_stats`, that's why I went with `pg_monitor`.
Also, I added a link to pre-defined role documentation.